### ISSUE TITLE
Добавить поддержку приточных установок Breezart.

### ISSUE THREAD

**Issue by uroopek** [sprut/Hub#312]
Добавить поддержку приточных установок Breezart, имеет все необходимое для полноценной [интеграции](http://www.breezart.ru/tech/breezart_smart_home.pdf) в умный дом. 

На форуме поддержки уже есть черновой вариант [транслятор](http://www.forum.breezart.ru/showthread.php?p=5641#post5641) в MQTT и на [Github](https://github.com/IronShaft/breezart-mqtt)

**Comment by MakeSimpleOrg** [comment]
Там свой протокол поверх HTTP, написание поддержки займет пару дней, и пока не имеет смысла.
Так же там есть модбас, это уже интереснее.

**Comment by uroopek** [comment]
Да, по сути Modbus тут самый интересный, с этим расчетом и пал выбор на данные приточные установки чтобы в дальнейшем завести в умный дом. 
Готов посодействовать и помочь чем смогу в том числе в виде благотворительности чтобы поддержка появилась.

**Comment by nick-lav** [comment]
поддерживаю, нужен breezart через modbus. вот описание регистров https://breezart.ru/tech/breezart_smart_home.pdf
В моем случае контроллер  JL204C7.
Регистры быстрого доступа
В контроллере есть регистры быстрого доступа Input[64000] – Input[64011], в которых хранятся
копии значений других регистров – это позволяет читать их одним Modbus запросом:
· Input[64000] = Holding[0]; // Заданная скорость вентилятора
· Input[64001] = Holding[1]; // Заданная температура
· Input[64002] = Holding[2]; // Заданная влажность
· Input[64003] = Holding[3]; // Состояние Вкл/Выкл
· Input[64004] = Holding[42]; // Опции управления пользовательские
· Input[64005] = Holding[7]; // Режим нагрев/охлаждение
· Input[64006] = Input[12005]; // Текущая загрязненность фильтра 0
· Input[64007] = Input[12205]; // Текущая загрязненность фильтра 1
· Input[64008] = Input[60000]; // Номер последнего сработавшего сценария
· Input[64009] = Input[22528]; // Статус внешнего увлажнителя (Carel)
· Input[64010] = Input[10010]; // Производительность вентилятора, %

**Comment by SubliminalFeelings** [comment]
Горячо поддерживаю.

Здесь карта регистров от производителя:
https://www.mautomatics.ru/product/a/download/?id=94&index=0

Есть решение для WB вот здесь:
https://support.wirenboard.com/t/shablon-dlya-mautomatics-jl204s5-breezart-550-lux/10534

Нужен какой-то кастомный шаблон, чтобы СХ увидел это устройство и начал с ним взаимодействовать.
Поскольку регистров очень много, скорее всего каждый сам для себя сможет собрать нужный набор.

**Comment by Batyushin** [comment]
вопросы подобного характера решаются в чате поддержки, необходимо предоставить топики устройства.
